konfera/models/organizer.py

```diff
@@ -0,0 +1,24 @@
+from konfera.models.countries import COUNTRIES
+from django.db import models
```

Comment:
`django.db` should be the first import and `.abstract` should be before `.countries`